sklearn/dummy.py

```diff
@@ -266,12 +267,12 @@ def predict_proba(self, X):
 
         Parameters
         ----------
-        X : {array-like, object with finite length or shape}
-            Training data, requires length = n_samples
+        X : array-like of shape (n_samples, n_features)
+            Test data.
 
         Returns
         -------
-        P : array-like or list of array-lke of shape (n_samples, n_classes)
+        P : array-like or list of array-like of shape (n_samples, n_classes)
```

Comment:
list is part of array-like so we can remove it

Reply:
Similar to `classes_`, `n_classes_` and `class_prior_`.